src/partials/catalog.html

```diff
@@ -1,3 +1,368 @@
-<header class="header">
-
-</header>
\ No newline at end of file
+<section class="catalog">
+    <div class="container">
+        <h2 class="section-header">Catalog</h2>
+        <ul class="catalog-list">
+            <li class="catalog-list-item">
+                <div class="img-wrap">
+                    <picture>
+                        <source srcset="./img/catalog/catalog-pic-1.webp 1x, ./img/catalog/catalog-pic-1@2x.webp 2x">
+                        <img class="catalog-list-img" src="./img/catalog/catalog-pic-1.png" width="335"
+                            alt="Tube Watch S42">
+                    </picture>
+                    <svg class="item-icon" width="24" height="24">
+                        <use href="../img/icons.svg#heart"></use>
```

Comment:
one dot should be in the address, as if you are now in index.html
in all below too